tests/test_config.py

```diff
@@ -11,8 +11,8 @@
 import itertools
 
 from grocker import __version__
-from grocker.__main__ import parse_config
-from grocker.helpers import default_image_name
+from grocker.utils import parse_config
+from grocker.utils import default_image_name
```

Comment:
All `-w-` commits will be merge with no `-w-` commit before pushing on master branch